app/config/config.yml

```diff
@@ -160,3 +162,12 @@ bazinga_geocoder:
             locale:  '%locale%'
             use_ssl: true
             api_key: '%env(GMAPS_API_KEY)%'
+
+snc_redis:
+    clients:
+        default:
+            type: predis
+            alias: default
+            dsn: "redis://%env(REDIS_HOST)%"
```

Comment:
Good idea to set the host in a parameter! This also allow an easy switch between docker and local Redis (in dev).